src/webui/src/components/PackageSidebar/index.jsx

```diff
@@ -0,0 +1,192 @@
+import React from 'react';
```

Comment:
I think we can break this component into smaller components. `MaintainerInfo`  and `Module` can be separated. @juanpicado @Meeeeow what do you think ?

Reply:
Yes, the `render` method is already long. We have to split it up. The 3 `<Module>` can be easily 3 components for each type.

Reply:
Furthermore, In order to create a smaller test. I usually follow this rule. 1 Module === 1 File